### PR TITLE
adding tests for Convert class methods

### PR DESCRIPTION
**Pull request by jiyoontbd**
# Overview
Added unit tests for `Convert.kt`

# How Has This Been Tested?
Ran unit tests locally and measured coverage. Now getting 98% coverage on Convert.kt

# Checklist

Before submitting this PR, please make sure:

- [x] I have read the CONTRIBUTING document.
- [x] My code is consistent with the rest of the project 
- [x] I have tagged the relevant reviewers and/or interested parties
- [x] I have updated the READMEs and other documentation of affected packages

### CODE REVIEW

File: common/src/test/kotlin/web5/sdk/common/ConvertTest.kt [hunk 1]
```diff
@@ -17,13 +19,199 @@ class ConvertTest {
...
+      val input = 1
+
+      assertFailsWith<UnsupportedOperationException> {
+        Convert(input, EncodingFormat.Base58Btc).toBase64Url()
+      }
+    }
+
+    @Test
+    fun toBase58Btc_Base58Kind() {
+      val input = "foo"
```
Comment (andresuribe87) [inline common/src/test/kotlin/web5/sdk/common/ConvertTest.kt:49]: If you're so inclined, here's a way in which we've parameterized tests in the past https://github.com/TBD54566975/web5-kt/blob/48ce6764163b32714be7d8282bfad0678f49debb/dids/src/test/kotlin/web5/sdk/dids/methods/web/DidWebTest.kt#L20

File: common/src/test/kotlin/web5/sdk/common/ConvertTest.kt [hunk 2]
```diff
@@ -17,13 +19,199 @@ class ConvertTest {
...
+      assertArrayEquals(input, output)
+    }
+
+    @Test
+    fun toByteArray_UnsupportedValueKind() {
+      val input = 1
+
+      assertFailsWith<UnsupportedOperationException> {
+        Convert(input).toByteArray()
+      }    }
```
Comment (andresuribe87) [inline common/src/test/kotlin/web5/sdk/common/ConvertTest.kt:215]:
```suggestion
      }
    }
```

**Review by andresuribe87 — APPROVED** [review]
Small nit; thanks!

**Review by mistermoe — APPROVED** [review]
🏅

**Review by tomdaffurn — APPROVED** [review]

**Pull request merged**